src/plugins/navigation/review/navigator.js

```diff
@@ -552,6 +559,14 @@ function navigatorFactory(config, map, context) {
         navigator.trigger('jump', position);
     }
 
+    if (config["reviewLayout"] === 'fizzy') {
```

Comment:
Are you sure the default layout is still working ok? I changed my value in `config/taoQtiTest/testRunner.conf.php` to `'not fizzy'`, and I get this strange rendering:
![Screenshot 2021-11-05 at 10 15 03](https://user-images.githubusercontent.com/43652944/140487280-80707ebf-72bf-406b-af8f-ba0a07e92c50.png)

Reply:
Yeah. Here my mistake. When i developed styles in separate file i import it after the `_navigator.scss`, so when layout switch back, it has rewritten styles. I'll change order and adopt new styles for this case.

Reply:
fixed